addons/calendar/models/calendar_attendee.py

```diff
@@ -124,9 +127,9 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
                         'datas': base64.b64encode(ics_file),
                         'description': 'invitation.ics',
                         'mimetype': 'text/calendar',
+                        'name': 'invitation.ics',
```

Comment:
Not really worth fixing if we're not touching that line tbh
Polutes the blaming process

Reply:
Actually it got added in the meantime by a different PR